sycl/source/detail/kernel_compiler/kernel_compiler_opencl.hpp

```diff
@@ -25,7 +25,7 @@ spirv_vec_t OpenCLC_to_SPIRV(const std::string &Source,
                              const std::vector<std::string> &UserArgs,
                              std::string *LogPtr);
 
-bool OpenCLC_Compilation_Available();
+bool OpenCLC_Compilation_Available(const std::vector<uint32_t> &IPVersionVec);
```

Comment:
Needs a comment. Do we check any_of or all_of here?

Reply:
the whole thing gets passed to ocloc as a -dev flag.  I presume it is all_of, but I don't know for sure.